assets/app/views/modals/debug-terminal.html

```diff
@@ -30,7 +30,7 @@ <h2 ng-if="debugPod.status.phase !== 'Failed'" class="text-muted">
     <div ng-if="containerState.running">
       <div class="help-block">
         This temporary pod has a modified entrypoint command to debug a failing container. The pod
-        will be deleted when the terminal is closed or after one hour.
+        will be deleted when the terminal window is closed.
```

Comment:
Maybe:
The pod will be available for one hour and will be deleted when the terminal window is closed.

Just hate for someone to run some long command trying to debug something and not realize the container will stop after an hour.